packages/mdc-checkbox/index.ts

```diff
@@ -23,4 +23,5 @@
 
 export * from './adapter';
 export * from './component';
+export * from './constants';
```

Comment:
for consistency, doesn't it make sense to do the `import * as` thing for all of them?

Reply:
ya we should be consistent. Good call!

Reply:
@abhiomkar actually makes a good point that the constants should be prefixed with `MDCx` like `MDCSelectFoundation`, etc. For consistency I will update these to `MDCSelectConstants`. So the result will be:

```
mdc.select.MDCSelectConstants.cssClasses;
```

Reply:
Curious, does re-exporting named alias might affect tree-shaking? For example, `MDCSelectConstants` bundles everything including cssClass, strings, numbers, etc?

Reply:
If it does bundle everything, I think it makes sense to export them as-is as @mmalerba suggested for simplicity.

Reply:
@abhiomkar what @mmalerba said in [this comment](https://github.com/material-components/material-components-web/pull/4593#discussion_r276853644), is to name space all of them. Is that what you're proposing to do?

Reply:
I just tested this out, it does look like merging them all into a single object prevents unused constants from being tree-shaken. I guess its kind of a similar problem as with tacking it all on to the Foundation class. 

To a certain degree it's a problem even if we don't merge them (e.g. using one of the `cssClasses` means none of the others can be tree shaken). I guess the ideal from a tree-shaking perspective would be if every constant was a separate export

Reply:
I agree that ideally every constant needs to be separate export in tree-shaking perspective.

Also it's worth noting that since we were not re-exporting separate constants from index file we ended up exposing them via Foundation's static methods. We might deprecate these static methods from Foundation for clean up.

Reply:
Hey picking this back up since I have less on my plate. If we were to export every constant individually, would we get rid of 'string', 'numbers', 'cssClasses'? And the entire constants file would be flat? I agree if we go through with this change, we should also remove the static members from foundation. This however will be a breaking change, so I think we'd have to keep them around with a deprecation notice.